### ISSUE TITLE
Minor typo in spec XR_EXT_hp_mixed_reality_controller

### ISSUE THREAD

**Issue by keithbradner** [KhronosGroup/OpenXR-SDK-Source#202]
The interaction profile path isn't plural. It's listed as:
```
/interaction_profile/hp/mixed_reality_controller
```
when it should be:
```
/interaction_profiles/hp/mixed_reality_controller
```

https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_EXT_hp_mixed_reality_controller

**Comment by rpavlik** [comment]
Thanks, a fix is coming in the next patch release.

**Comment by rpavlik** [comment]
Fixed in 1.0.11
